.github/workflows/main.yml

```diff
@@ -8,6 +8,8 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
+        # oldest and newest supported LLVM version
+        clang_version: [10.0.0, 14.0.0]
```

Comment:
In principle: Yes. But then I would have to copy-paste the whole Linux-section (for examle) and just change the version. I thought this would be better maintainable. Also, if it's the minimal supported LLVM-version, then it should build on all platforms, so I thought it a good idea to test all (even though chances are low it will break on only one. But who knows?). 
Do you want me to change it?

Reply:
No, this seems reasonable as is.  If CI times become a problem we can look into shrinking the testing matrix